code/script.js

```diff
@@ -0,0 +1,201 @@
+
+        // Last Commit Message - Committed Netlify Link to add to live link.
+        let lastCommitLink = data[0].commit.message
+
+        // Formatting the DATES
+        // Last Commit Date which is NOT the Netlify commit (Index 1 instead of 0)
+        // Formatting DAYS
+        let lastCommitDateRaw = data[1].commit.committer.date.substring(0,10)
+        // Removing the 0 from dates before the 10th for DAYS
+        let lastCommitDayRaw = lastCommitDateRaw.substring(8,10)
+        let lastCommitDay
+        if (lastCommitDayRaw < 10) {
+            lastCommitDay = lastCommitDayRaw.replace("0", "")
+        } else {
+            lastCommitDay = lastCommitDayRaw;
+        }
+        // Formatting MONTHS
+        const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+        let lastCommitMonthNumber = lastCommitDateRaw.substring(5,7).replace("0","")
+        let lastCommitMonth = months[lastCommitMonthNumber - 1]
+        // Extracting YEAR
+        let lastCommitYear = lastCommitDateRaw.substring(0,4)
+
+        let commitDate = `${lastCommitMonth} ${lastCommitDay}, ${lastCommitYear}`
+
+        // Creating the Project Elements
+        document.getElementById(`${projectsID}2`).innerHTML += `
+                <p>Number of commits: ${numberOfCommits}</p>
+                <p>Last commit date: ${commitDate}</p>
+            `
+
+        // If statement to check if last commit includes netlify link
+        if (lastCommitLink.includes('netlify') === true) {
+            document.getElementById(`${projectsID}2`).innerHTML += `
+            <p>View it live <a href="${lastCommitLink}" target="_blank">here</a>.
+            `
+        } else {
+            document.getElementById(`${projectsID}2`).innerHTML += `
+            <p>No live link available.</p>
+            `
+        }
+    })
+}
+
+// Fetching the pull requests
+const pullFetch = (projectName, projectsID) => {
+    const GIT_FETCH_API = `https://api.github.com/repos/technigo/${projectName}/pulls?per_page=100`
+    fetch(GIT_FETCH_API, options)
+        .then(res => res.json())
+        .then(data => {
+
+        // Filter Pull Requests by my username
+        const filteredPR = data.filter(function (pr) {
+            return pr.user.login === username
+        })
+
+        // Check if there is a pull request with my username
+        let exist = false
+        for (let i = 0; i < data.length; i++) {
+            if (data[i].user.login === username) {
+                exist = true
+            }
+        }
+
+        // Conditional for pull request innerHTML
+        if (exist === true) {
+            document.getElementById(`${projectsID}2`).innerHTML += `
+            <p>Pull request with comments <a href="${filteredPR[0].html_url}" target="_blank">here</a>.</p>
+            `
+        } if (exist === false) {
+            document.getElementById(`${projectsID}2`).innerHTML += `
+            <p>Pull request unavailable.</p>
```

Comment:
Nice solution to be able to display that there isn't a PR! Good for the user to see and not have to wonder where the PR link is:)